packages/vite/LICENSE.md

```diff
@@ -535,28 +535,6 @@ License: MIT
 By: Rich Harris
 Repository: rollup/plugins
 
-> The MIT License (MIT)
```

Comment:
Why is the full license no longer needed here? 🤔

Reply:
It seems it no longer exists since 4.1.1.

https://www.jsdelivr.com/package/npm/@rollup/pluginutils?version=4.1.0
https://www.jsdelivr.com/package/npm/@rollup/pluginutils?version=4.1.1
https://www.jsdelivr.com/package/npm/@rollup/pluginutils

Reply:
I would expect each package in the official plugin monorepo to still be released with a LICENSE file. @lukastaegert is this intentional? Are we ok here without including a license after we bundle pluginutils?

Reply:
That is a good point. I think this was not considered at that point. There is a central license in the plugins repo, but unfortunately the license is no longer distributed with the packages. Which is really stupid is the distribution of the packages is validating their own license. Rest assured, this is still MIT licensed.

Reply:
Totally. My question is if Vite is doing things right after this PR. If we can bundle the package and remove the license text from our aggregated license file. It looks to me that these packages should have the license file, or we need to update the license extractor